core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -222,7 +234,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("consumer_prop")
       .ofType(classOf[String])
-    val consumerConfigOpt = parser.accepts("consumer.config", s"Consumer config properties file. Note that ${consumerPropertyOpt} takes precedence over this config.")
+    val consumerConfigOpt = parser.accepts("consumer.config", "Consumer config properties file.")
```

Comment:
That's my miss after recently rebasing the PR. Thanks for pointing it out. I'll revert it back.